### ISSUE TITLE
Federation/ActivityPub

### ISSUE THREAD

**Issue by ThatOneCalculator** [Goldfish-Social/Goldfish#1]
Will Goldfish be federated, and if so, will it be on ActivityPub?

**Comment by mstdn** [comment]
> Will Goldfish be federated, and if so, will it be on ActivityPub?

Yes and yes! That's the idea. Pixelfed offers a great example with this
